### PR TITLE
 add Cache-Control header(365day) for S3 upload

### PR DESCRIPTION
**Pull request by naofumi-fujii**
🙏

### CODE REVIEW

File: config/initializers/carrierwave.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@
     }
     config.asset_host    = ENV["ASSET_HOST"]
     config.fog_directory = ENV["S3_BUCKET"]
+    config.fog_attributes = { 'Cache-Control' => "max-age=#{365.day.to_i}" }
```
Comment (tricknotes) [inline config/initializers/carrierwave.rb]: `day` -> `days` ?

**Comment by tricknotes** [comment]
Thanks @naofumi-fujii !
This PR is very awesome.

File: config/initializers/carrierwave.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@
     }
     config.asset_host    = ENV["ASSET_HOST"]
     config.fog_directory = ENV["S3_BUCKET"]
+    config.fog_attributes = {"Cache-Control" => "max-age=#{365.days.to_i}"}
```
Comment (naofumi-fujii) [inline config/initializers/carrierwave.rb:13]: changed day -> days 🌟

**Comment by tricknotes** [comment]
⭐️

**Pull request merged**